library/src/main/java/com/datatorrent/lib/io/fs/FSInputModule.java

```diff
@@ -57,6 +58,8 @@
   private long blockSize;
   private boolean sequencialFileRead = false;
   private int readersCount;
+  @Min(1)
+  protected int blocksThreshold;
```

Comment:
why not initialize it to 1 so that user don't have to set anything if user goes with default value